code/espurna/config/dependencies.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DEPENDENCIES_H
```

Comment:
Just as sensors this should be pragma once.

Reply:
good point, changed.